test/sensor/camera/test_projection.py

```diff
@@ -0,0 +1,44 @@
+import torch
+
+from kornia.geometry.vector import Vector2, Vector3
+from kornia.sensor.camera.projection import Z1Projection
+from kornia.testing import BaseTester
+
+
+class TestZ1Projection(BaseTester):
```

Comment:
And here, can we have a compile test here too? 😄 

and skip the unnecessary tests